### PR TITLE
Us 12306 : inset component, by string manipulation

### PR DESCRIPTION
**Pull request by SushreeM-h**

### CODE REVIEW

**Comment by SushreeM-h** [comment]
12030 pr was revereted so no need of fixes.

File: src/components/override-sdk/template/DefaultForm/DefaultFormUtils.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  const textToBeInserted = `<div class="govuk-warning-text">
+          <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+          <strong class="govuk-warning-text__text">
+            <span class="govuk-warning-text__assistive">WARNING</span>
+            ${textToBeFormatted}
+          </strong>
+        </div>`;
+
+  finalText = finalText.replaceAll(textToBeReplaced, textToBeInserted);
+  return finalText;
```
Comment (gautamkumar-hmrc) [inline src/components/override-sdk/template/DefaultForm/DefaultFormUtils.tsx:13]: return finalText.replaceAll(textToBeReplaced, textToBeInserted);

File: src/components/override-sdk/template/DefaultForm/DefaultFormUtils.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+}
+
+function replaceInsetText(finalText, textToBeReplaced, textToBeFormatted) {
+  const textToBeInserted = `<div class="govuk-inset-text">${textToBeFormatted}</div>`;
+  finalText = finalText.replaceAll(textToBeReplaced, textToBeInserted);
+  return finalText;
```
Comment (gautamkumar-hmrc) [inline src/components/override-sdk/template/DefaultForm/DefaultFormUtils.tsx:19]: same as above

File: src/components/override-sdk/template/DefaultForm/DefaultForm.tsx [hunk 3]
```diff
@@ -44,9 +44,11 @@ export default function DefaultForm(props) {
   }
   const instructionExists = instructionText !== undefined && instructionText !== '';
 
-  const settingTargetForAnchorTag = () => {
+  const isOnlyFieldDetails = useIsOnlyField();
+
+  function settingTargetForAnchorTag() {
     const instructionDiv = document.getElementById('instructions');
-    const keyText = t('OPENS_IN_NEW_TAB');
+    const keyText = 'OPENS_IN_NEW_TAB';
```
Comment (gautamkumar-hmrc) [inline src/components/override-sdk/template/DefaultForm/DefaultForm.tsx:51]: is this intentional?

**Review by AkhilaHMRC — APPROVED** [review]

**Pull request merged**